test/rest.middleware.test.js

```diff
@@ -164,6 +164,40 @@ describe('loopback.rest', function() {
     }, done);
   });
 
+  it('should report 200 for legacy explorer route /routes', function(done) {
+    app.model(MyModel);
```

Comment:
Is it necessary to add `MyModel` to the app? It's better to keep the test as small as possible and remove anything not needed. The same comment applies to the other new tests.